components/payloads/cu_ros_payloads/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "cu-ros-payloads"
+version.workspace = true
+authors.workspace = true
+edition.workspace = true
+license.workspace = true
+keywords.workspace = true
+categories.workspace = true
+homepage.workspace = true
+repository.workspace = true
+description = "Those are ROS compatible payloads"
```

Comment:
This will be the basic public description on Crates.io, maybe mention that it is part of copper/extra payload for copper.